modules/ROOT/pages/functions/temporal/duration.adoc

```diff
@@ -16,10 +16,11 @@ See also xref::values-and-types/temporal.adoc[Temporal values] and xref::express
 
 .Details
 |===
-| *Syntax* 3+| `duration(input)`
+| *Syntax* 3+| `duration(input[, pattern])`
 | *Description* 3+| Creates a `DURATION` value.
-.2+| *Arguments* | *Name* | *Type* | *Description*
+.3+| *Arguments* | *Name* | *Type* | *Description*
 | `input` | `ANY` | A map optionally containing the following keys: 'years', 'months', 'weeks', 'days', 'hours', 'minutes', 'seconds', 'milliseconds', 'microseconds', or 'nanoseconds'.
+| `pattern` | `STRING` | A pattern used to parse the input. If a pattern is provided, `input` must be `STRING`.
```

Comment:
This is correct, but the description of the line above sounds like it always need to be a map. I think we need to rephrase that so it is clearer that it can be a string or a map